tests/e2e/tests/01_EmbedStreamTest.js

```diff
@@ -0,0 +1,44 @@
+export default {
```

Comment:
We shouldn't be mixing the import styles on FE JS and BE JS. I'd prefer to use the `module.exports = ` syntax here.